lesson-4/Chen CSS- Lesson 4/style.css

```diff
@@ -0,0 +1,55 @@
+body {
+	color:gold;
+}
+
+h1 {
+	font-family: minion, arial, upper-cut;
+	text-align:center;
+}
+
+h2 {
+	color:navajowhite;
+	font-size: 1em;
+}
+
+ul {
+	list-style-type: square;
+}
+
+ol {
+	list-style-type: upper-alpha;
+}
+
+li{font-size: 1em;
+}
+
+
+div p {
+	background-color: pink;
+	font-size: 2em;
+}
+
+.special-header{
+	color:blue;
+}
+
+#what-is-css {
+	background-color: hotpink;
+}
+
+em{
+	font-size:3rem;
+	font-style:bold;
+}
+
+.space{
+	border-radius: 50px;
+	height:200px;
+	width:200px;
+}
+#blue{border:4px solid dodgerblue;	
```

Comment:
The way you've formatted your CSS in the above sections, where the curly brackets (`{` `}`) are on separate lines is easier to read. Try to keep it consistent for these bottom 3 as well!